### PR TITLE
remove long click download action

### LINKED ISSUE #2868: minor improvements to media viewer
I would like to remove the download on long click action because it conflicts with a really nice android gesture (double tap and drag up and down to zoom) and I would also like to add an action to download media to a specific folder.

@bmarty any opinions on the two changes before I start working on them?

### PR DESCRIPTION
**Pull request by dkanada**
This fixes #2868 and reduces duplicate code.

### CODE REVIEW

File: vector/src/main/java/im/vector/activity/VectorMediaViewerActivity.java [hunk 1]
```diff
@@ -338,9 +338,6 @@ public void onRequestPermissionsResult(int requestCode, @NonNull String[] permis
             if (requestCode == PermissionsToolsKt.PERMISSION_REQUEST_CODE) {
                 // Request comes from here
                 onAction(mPendingPosition, mPendingAction);
-            } else if (requestCode == PermissionsToolsKt.PERMISSION_REQUEST_OTHER) {
```
Comment (bmarty) [inline vector/src/main/java/im/vector/activity/VectorMediaViewerActivity.java]: `PERMISSION_REQUEST_OTHER` could be removed too.

Comment (dkanada) [reply vector/src/main/java/im/vector/activity/VectorMediaViewerActivity.java]: Do you mean `PERMISSION_REQUEST_CODE` in the line above? I think that might be used for the normal download action.

Comment (bmarty) [reply vector/src/main/java/im/vector/activity/VectorMediaViewerActivity.java]: No, I mean removing `PERMISSION_REQUEST_OTHER` from file `PermissionsTools.kt`, sorry, I was not clear

**Comment by dkanada** [comment]
@bmarty should be good.

**Comment by bmarty** [comment]
Yes, thanks

**Pull request merged**